docs/spec/WIP_provisioning/overview.md

```diff
@@ -0,0 +1,229 @@
+# Fee Distribution
+
```

Comment:
Can we add a brief "Motivation" section (reducing necessary calculation)

Reply:
Not in this PR - next PR this is only staking